release.json

```diff
@@ -29,25 +29,25 @@
         "SECURITY_AGENT_POLICIES_VERSION": "master"
     },
     "release-a6": {
-        "INTEGRATIONS_CORE_VERSION": "7.47.0-rc.3",
-        "OMNIBUS_SOFTWARE_VERSION": "7.47.0-rc.1",
-        "OMNIBUS_RUBY_VERSION": "7.47.0-rc.1",
+        "INTEGRATIONS_CORE_VERSION": "7.48.0-rc.1",
+        "OMNIBUS_SOFTWARE_VERSION": "7.48.0-rc.1",
+        "OMNIBUS_RUBY_VERSION": "7.48.0-rc.1",
         "JMXFETCH_VERSION": "0.47.10",
         "JMXFETCH_HASH": "8625f661e34b7180e9df726c69f9edcf71c82682510226948e8a3b37c45c4b9d",
         "SECURITY_AGENT_POLICIES_VERSION": "v0.48.0",
-        "MACOS_BUILD_VERSION": "6.47.0-rc.1",
+        "MACOS_BUILD_VERSION": "6.48.0-rc.1",
         "WINDOWS_DDNPM_DRIVER": "release-signed",
         "WINDOWS_DDNPM_VERSION": "2.4.1",
         "WINDOWS_DDNPM_SHASUM": "f12af44306eac3ea15828fd12c24d44ae519692a94a0f1f5d4fa868c3e596b07"
     },
     "release-a7": {
-        "INTEGRATIONS_CORE_VERSION": "7.47.0-rc.3",
-        "OMNIBUS_SOFTWARE_VERSION": "7.47.0-rc.1",
-        "OMNIBUS_RUBY_VERSION": "7.47.0-rc.1",
+        "INTEGRATIONS_CORE_VERSION": "7.48.0-rc.1",
+        "OMNIBUS_SOFTWARE_VERSION": "7.48.0-rc.1",
+        "OMNIBUS_RUBY_VERSION": "7.48.0-rc.1",
         "JMXFETCH_VERSION": "0.47.10",
```

Comment:
should we update JMXFetch version too?

Reply:
This version was set one week ago in https://github.com/DataDog/datadog-agent/pull/18735 with added release notes and 7.48.0 milestone. I believe this is ok then

Reply:
And it was confirmed by AML team